Benv547/cma-back-application
# feat: add bootstrap app

Ticket: https://trello.com/c/hvomrVSi

## What's Changed

- Introduced a new boilerplate code.

## TODO 

- [x] Update mvn with packages

## Dependencies PRs

- None

## How to test
- Run the test to check if application work with the following command: ```mvn test```
